tests/js/sentry-test/reactTestingLibrary.tsx

```diff
@@ -60,12 +49,14 @@ function makeAllTheProviders({context, organization}: ProviderOptions) {
  * After
  * mountWithTheme(<Something />, {context: routerContext});
  */
-function mountWithTheme(ui: React.ReactElement, options?: Options) {
-  const {context, organization, ...otherOptions} = options ?? {};
+function mountWithTheme(component: React.ReactElement, options: Options = {}) {
+  const {context, ...testingLibraryOptions} = options;
 
-  const AllTheProviders = makeAllTheProviders({context, organization});
+  const BaseProviders = makeBaseProviders(context);
```

Comment:
I would probably also opt to remove the legacy context here too, then our mountWithTheme is even closer to what render does

Reply:
We still use legacy context in a few places (react router I think) but we should stop using it for organization context if we haven't already.

Reply:
@markstory Yeah, that's a very good point. I think given that we would phase those out, it would be worth evaluating what it would take to either migrate away from those directly and update the components, or check how many tests would have to be updated. That way it's not part of the API going forward, and we need to manually set it for each test. I have actually seen some tests where we were using RouterContext with default Organization and were then passing an Organization with different attributes to the tested component which resulted in a weird test and kind of "undefined" behaviour